extensions/amp-story-panning-media/validator-amp-story-panning-media.protoascii

```diff
@@ -17,4 +17,23 @@ tags: {  # <amp-story-panning-media>
   amp_layout: {
     supported_layouts: FILL
   }
+  attrs: {
+    name: "x"
```

Comment:
have you considered using `data-` attributes in order to reduce the possibility of a collision with a browser attribute in the future? Short attributes like this seem more likely than average to cause problems down the line.

Reply:
Thanks for mentioning this Greg! Agreed that it's a good idea to add them. Updated.